src/components/customer-resource-show.js

```diff
@@ -83,6 +83,14 @@ export default function CustomerResourceShow({ model, toggleSelected }) {
 
               <hr/>
 
+              {model.visibilityData.isHidden && (
+                <div data-test-eholdings-customer-resource-show-is-hidden>
+                  <p><strong>This resource is hidden.</strong></p>
+                  <p><em>{model.visibilityData.reason}</em></p>
+                  <hr />
+                </div>
+              )}
```

Comment:
This will need to be adjusted to use `resource` instead of `model` once you rebase onto master